ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -498,6 +511,29 @@ export default class ConfirmTransactionBase extends Component {
     return <ConfirmData txData={txData} dataComponent={dataComponent} />;
   }
 
+  ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+  renderNote() {
+    const { t } = this.context;
+    const { isNoteToTraderSupported } = this.props;
+    if (!isNoteToTraderSupported) {
+      return null;
+    }
+    return (
+      <Box className="confirm-page-container-content__data">
+        <Box display={DISPLAY.FLEX} flexDirection={FLEX_DIRECTION.ROW}>
+          <NoteToTrader
+            maxLength="280"
+            placeholder={t('notePlaceholder')}
+            onChange={(value) => this.setState({ noteText: value })}
```

Comment:
@jpuri could you review this and see how it fits into your plans for confirmation team/refactors?

Reply:
Thanks a lot @brad-decker , this class ConfirmTransactionBase has become very large due to code being added to it like above. We are refactoring this to remove code like `renderNote` into separate components. I would suggest moving this component outside `ConfirmTransactionBase`.